packages/web-react/src/components/PricingPlan/PricingPlan.tsx

```diff
@@ -0,0 +1,38 @@
+'use client';
+
+import classNames from 'classnames';
+import React, { ElementType } from 'react';
+import { useStyleProps } from '../../hooks';
+import { SpiritPricingPlanProps } from '../../types/pricingPlan';
+import { mergeStyleProps } from '../../utils';
+import { NUMBER_OF_PLAN_ROWS_DEFAULT } from './constants';
+import { usePricingPlanStyleProps } from './usePricingPlanStyleProps';
+
+const defaultProps: Partial<SpiritPricingPlanProps> = {
+  isComparable: false,
+  isHighlighted: false,
+  rows: NUMBER_OF_PLAN_ROWS_DEFAULT,
+};
+
+const PricingPlan = <T extends ElementType = 'article'>(props: SpiritPricingPlanProps<T>) => {
+  const propsWithDefaults = { ...defaultProps, ...props };
+  const { elementType: ElementTag = 'article', children, ...restProps } = propsWithDefaults;
+
+  const { classProps, props: modifiedProps, styleProps: pricingPlanStyleProps } = usePricingPlanStyleProps(restProps);
+  const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
+  const mergedStyleProps = mergeStyleProps(ElementTag, {
+    classProps: classProps.root,
+    pricingPlanStyleProps,
+    styleProps,
+  });
+
+  return (
+    <ElementTag {...otherProps} {...mergedStyleProps}>
```

Comment:
Since you can change the element type of `PricingPlan`, it makes sense to enable changing it also for `PricingPlanHeader`, `PricingPlanBody`, and `PricingPlanFooter`. I think it's fine to do it later if you don't find time for it now, please just file an issue for that.